dev_settings.py

```diff
@@ -16,6 +16,8 @@
 
 TEST_RUNNER = 'testrunner.DevTestRunner'
 
+SKIP_TESTS_REQUIRING_EXTRA_SETUP = True
```

Comment:
Some devs will not have external dependencies like kafka or elasticsearch running. This will cause tests that require those dependencies to be skipped rather than failing hard. Note: it's possible to see why a test was skipped by increasing the verbosity when running tests:
```
$ ./manage.py --settings=testsettings -v2 corehq/apps/reports/tests/test_esaccessors.py
...
test_inactive_user_query (corehq.apps.reports.tests.test_esaccessors.TestUserESAccessors) ... SKIP: ConnectionError: ConnectionError(HTTPConnectionPool(host='localhost', port=9200): Max retries exceeded with url: /test_hqusers_2015-11-12_1638 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10b0d6390>: Failed to establish a new connection: [Errno 61] Connection refused',))) caused by: MaxRetryError(HTTPConnectionPool(host='localhost', port=9200): Max retries exceeded with url: /test_hqusers_2015-11-12_1638 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10b0d6390>: Failed to establish a new connection: [Errno 61] Connection refused',)))
```